hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -293,6 +295,8 @@ public class OmMetadataManagerImpl implements OMMetadataManager,
   private Map<String, Table> tableMap = new HashMap<>();
   private List<TableCacheMetrics> tableCacheMetrics = new LinkedList<>();
 
+  private TableCache<CacheKey<String>, CacheValue<S3SecretValue>> s3SecretCache;
```

Comment:
I don't think the secret cache needs to be coupled with the secret store implementation. Just using the `S3ImMemoryCache` for both caches is good enough. At the end of the day, they're just im-memory cache.